hedera-node/hedera-app/src/test/resources/bootstrap.properties

```diff
@@ -90,7 +90,7 @@ expiry.throttleResource=expiry-throttle.json
 contracts.redirectTokenCalls=true
 contracts.referenceSlotLifetime=31536000
 contracts.scheduleThrottleMaxGasLimit=5000000
-contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
```

Comment:
Is this the right file? I would think we'd need to add this line to `hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties`

Reply:
Should already exist there https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties#L94

Reply:
It's already there.  To the best of my knowledge there are two kinds of bootstrap files: _Ur_-bootstrap files which contain _every_ property (and _must_ contain _something_ for every property) and _delta_ bootstrap files (my name) which contain overrides.  This is an _ur_-bootstrap file so it needs this property.  AFAIK.  TBH, this is a rather _new_ file with the rather _new_ `hedera-app` - I don't know under what circumstances it is used.